plugins/modules/site_workflow_manager.py

```diff
@@ -760,6 +1088,47 @@ def get_want(self, config):
             parameters such as 'site_params' and 'site_name.' The gathered
             information is stored in the 'want' attribute for later reference.
         """
+        try:
+            type_info = self.config[0]['site']['type']
+            want_list = []
+
+            if type_info == "bulk":
+                if self.dnac_version >= self.version_2_3_7_6:
+                    self.keymap = {}
+                    self.keymap = self.map_config_key_to_api_param(
+                        self.keymap, config)
+                    self.keymap.update({
+                        "floor_number": "floorNumber",
+                        "rf_model": "rfModel",
+                        "parent_name_hierarchy": "parentNameHierarchy",
+                        "units_of_measure": "unitsOfMeasure"
+                    })
+                    self.log(self.keymap, "INFO")
+                else:
+                    self.log(
+                        "In this version '{0}' Bulk site operation file is not supported.".format(
+                            self.payload.get("dnac_version")), "ERROR"
+                    )
+                    self.module.fail_json(msg="In this version '{0}' Bulk site operation file is not supported.".format(
```

Comment:
Avoid using fail_json and instead of this set the status to failed and set the failed message in self.msg and invoke check_return_status API.

Reply:
Modified